v-next/hardhat/src/internal/builtin-plugins/node/helpers.ts

```diff
@@ -0,0 +1,58 @@
+import type { EdrNetworkAccountsConfig } from "../../../types/config.js";
+
+import { hexStringToBytes } from "@ignored/hardhat-vnext-utils/hex";
+import chalk from "chalk";
+import { addr } from "micro-eth-signer";
+
+import { isDefaultEdrNetworkHDAccountsConfig } from "../network-manager/edr/edr-provider.js";
+import { normalizeEdrNetworkAccountsConfig } from "../network-manager/edr/utils/convert-to-edr.js";
+
+export async function printEdrNetworkConfigAccounts(
+  config: EdrNetworkAccountsConfig,
+): Promise<void> {
+  const accounts = await normalizeEdrNetworkAccountsConfig(config);
```

Comment:
@alcuadrado this will need to be resolved differently if we move the derivation process to EDR

Reply:
Can you add it to your issue?

Reply:
good point btw, I hadn't noticed that

Reply:
Actually, this is an interactive task that prints to the console (printing is slow), where 200-300ms may not matter.

Reply:
Agreed, but seems weird to keep the code in the frontend just for this logic. Btw the issue to move the derivation to EDR was going to be created by @nebasuke